src/parse-stream.js

```diff
@@ -624,6 +647,40 @@ export default class ParseStream extends Stream {
         return;
       }
 
+      match = (/^#EXT-X-I-FRAME-STREAM-INF:(.*)$/).exec(newLine);
+      if (match) {
+        event = {
+          type: 'tag',
+          tagType: 'i-frame-playlist'
+        };
+
+        event.attributes = parseAttributes(match[1]);
+
+        if (event.attributes.URI) {
+          event.uri = event.attributes.URI;
+        }
+
+        if (event.attributes.BANDWIDTH) {
+          event.attributes.BANDWIDTH = parseInt(event.attributes.BANDWIDTH, 10);
+        }
+
+        if (event.attributes.RESOLUTION) {
+          event.attributes.RESOLUTION = parseResolution(event.attributes.RESOLUTION);
```

Comment:
Can you also replace other resolution parsing places with this function?

Reply:
@dzianis-dashkevich thanks for the review!

My first thought was to create a separate PR, as this wasn't the scope of this PR. But if you agree that this PR should include a refactoring commit, I'm all for it. Let me know what you prefer.

I'll do that a little later today.

Reply:
> My first thought was to create a separate PR, as this wasn't the scope of this PR
This is always a good approach, but I believe it is a pretty small change, and it will be visually easier to review here.

If you feel differently about it, I am absolutely fine with a separate PR

Thank you!